0001-typedb/02-dev/05-read.md

```diff
@@ -0,0 +1,401 @@
+---
+pageTitle: Reading data
+keywords: typeql, query, match, pattern, read
+longTailKeywords: typeql match, match get, typeql read, typedb read
+Summary: Reading data from a TypeDB database.
+---
+
+# Reading data
+
+There is only one query type to retrieve data instances and types from a database: get query.
+
+To try the following examples use [TypeDB Studio](../../02-clients/01-studio.md) or other TypeDB 
+[Clients](../../02-clients/00-clients.md).
+
+For those developing applications with [TypeDB drivers](../../02-clients/00-clients.md#typedb-drivers), please see the 
+instructions and examples of how to send a query for a specific language/framework: 
+[Java](../../02-clients/03-java.md), 
+[Node.js](../../02-clients/05-nodejs.md), 
+[Python](../../02-clients/04-python.md).
+
+## Get query
+
+A get query triggers a search in a database based on what has been described in the `match` clause. It is 
+used to extract concepts (data instances or types) out of a database. The result of a get query can be quite different, 
+depending on the query itself ([modifiers](#modifiers), [aggregation](#aggregation), [grouping](#group)) 
+and transaction options ([inference](06-infer.md#inferring-data)). For more information see the 
+[Response interpretation](07-response.md) page.
+
+### Syntax
+
+Get queries are written in TypeQL with the following syntax:
+
+```bash
+match <pattern>
+[get <variable> [(, <variable>)...];] 
+[sort <variable> [asc|desc];] 
+[offset <value>;] 
+[limit <value>;] 
+[group <variable>;] 
+[count;] [sum|max|min|mean|median|std <variable>;]
```

Comment:
That's a bit overloaded. Should we separate this into get, get-aggregate, get-group, and get-group-aggregate? Not sure that'd be clearer, but worth a shot. After all, that is how they're represented in the clients.